### PR TITLE
:memo: doc

### PR DESCRIPTION
**Pull request by matyo91**

### CODE REVIEW

File: GDD.md [hunk 1]
```diff
@@ -60,18 +61,19 @@ This state allows the player to see hidden objects, obstacles, etc..
...
-TBD
+- Trampoline
 
 ## Game Controls
 | Action | Input | Description |
 | ------ | ----- | ----------- |
-| player_move_left | Keyboard key: `A` | Move player to the left... |
-| player_move_right  | Keyboard key: `S` | Move player to the right... |
-| player_jump  | Keyboard key: `W` | Jump, press twice to doulble jump... |
+| player_move_left | Keyboard key: `left` | Move player to the left... |
```
Comment (btzr-io) [inline GDD.md]: We can assign both keys to input actions.

Comment (btzr-io) [reply GDD.md]: So it can be: `player_jump: W, UP, etc...`

**Review by btzr-io — APPROVED** [review]
Looks good to me, are you going to add something else ?
Otherwise feel free to merge.

**Comment by matyo91** [comment]
I first wanted to make more vocabulary with "corruption", roots, portals, but prefer to keep it simple as you did.

**Pull request merged**